### DISCUSSION TITLE
Consistent "picture-in-picture" toggle before and after switching to fullscreen

### DISCUSSION THREAD

**Discussion by amezhong1** [sebdelsol/sfvip-all#14] [Ideas]
Currently if I switch to "picture-in-picture" mode, and switch on fullscreen, then switch off fullscreen, it goes back to regular windowed mode. Can the "picture-in-picture" toggle be made to be consistent before and after changing between fullscreen and windowed? Thank you!

**Comment by sebdelsol** [comment]
Please raise an issue in the relevant [**repo**](https://github.com/K4L4Uz/SFVIP-Player/issues/17).

**Comment by sebdelsol** [comment]
It has been fixed in _Sfvip Player_ v1.2.7.72. Please update it.
